src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
```

Comment:
I choose to keep `DocumentationServices` a `static` class for now, otherwise that will bring the rest of the world into this PR. With that decision made, `IPathManager` needs to be passed into the methods to help with path resolution.